chains/xcm-integration-tests/emulated/networks/dancelight-system/src/lib.rs

```diff
@@ -0,0 +1,92 @@
+// Copyright (C) Moondance Labs Ltd.
+// This file is part of Tanssi.
+
+// Tanssi is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+
+// Tanssi is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU General Public License for more details.
+
+// You should have received a copy of the GNU General Public License
+// along with Tanssi.  If not, see <http://www.gnu.org/licenses/>.
+
+pub use xcm_emulator::{bx, TestExt};
+use {
+    dancelight_emulated_chain::Dancelight,
+    frame_support::parameter_types,
+    frontier_template_emulated_chain::FrontierTemplate,
+    simple_template_emulated_chain::SimpleTemplate,
+    sp_keyring::Sr25519Keyring,
+    tanssi_emulated_integration_tests_common,
+    xcm_emulator::{
+        decl_test_networks, Bridge, BridgeLaneId, BridgeMessage, BridgeMessageDispatchError,
+        BridgeMessageHandler, Chain, Network,
+    },
+};
+
+pub struct DancelightEthMockBridge;
+impl Bridge for DancelightEthMockBridge {
+    type Source = DancelightRelay;
+    type Target = ();
+    type Handler = DancelightEthMockBridgeHandler;
+
+    fn init() {
+        <DancelightRelay as Chain>::Network::init();
+    }
+}
+
+decl_test_networks! {
+    pub struct DancelightMockNet {
+        relay_chain = Dancelight,
+        parachains = vec![
+            FrontierTemplate,
+            SimpleTemplate,
+        ],
+        bridge = DancelightEthMockBridge
+    }
+}
+
+parameter_types! {
+    // Dancelight
+    pub DancelightSender: dancelight_runtime::AccountId = Sr25519Keyring::Alice.to_account_id();
+    pub DancelightReceiver: dancelight_runtime::AccountId = Sr25519Keyring::Bob.to_account_id();
+    pub DancelightEmptyReceiver: dancelight_runtime::AccountId = DancelightRelay::account_id_of(tanssi_emulated_integration_tests_common::accounts::RANDOM);
+
+    // SimpleTemplate
+    pub SimpleTemplateSender: container_chain_template_simple_runtime::AccountId = dancelight_runtime::AccountId::from(tanssi_emulated_integration_tests_common::accounts::ALICE);
```

Comment:
```suggestion
    pub SimpleTemplateSender: container_chain_template_simple_runtime::AccountId = container_chain_template_simple_runtime::AccountId::from(tanssi_emulated_integration_tests_common::accounts::ALICE);
```

Reply:
And here we might not need to incorporate `dancleight_runtime` and other runtimes, I think we can do `AccountIdOf<Dancelight::Runtime>`

Reply:
I did it via this approach: `container_chain_template_simple_runtime::AccountId`.
If you share the example on how it can be done with `AccountIdOf<Dancelight::Runtime>`, I can replace it. But I didn't find a way, we would need to pull Dancelight Runtime I guess to the dependencies.